.gitattributes

```diff
@@ -1 +1 @@
-text eol=lf
+* text eol=lf
```

Comment:
isn't better to leave this at `autocrlf` such that correct line endings are checked out based on client OS, but only lf is used when it's pushed back?

Reply:
@cengique , could you test with using docker and mingw on windows to build?  Emeth was having difficulty before related to the line endings and building with docker on windows.

Reply:
I will try this.

Reply:
@dGr8LookinSparky can't get the docker build. I'll open a new issue for it. We can let this go for now since Windows builds are done through Linux Docker containers.

Reply:
@dGr8LookinSparky got the Docker build on Windows. Updated the README accordingly. Leaving the `.gitattributes` as is, but added a line to stop from README being converted to Unix line endings. Still, it changed the whole README, not sure what is happening with it.